dev-python/backports-shutil_which/backports-shutil_which-3.5.1.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
It is recommended (but not required as EAPI=5 is still acceptable ) to use the latest EAPI (6) when adding new packages.